launcher/src/main/java/org/wildfly/core/launcher/AbstractCommandBuilder.java

```diff
@@ -56,10 +56,16 @@
         DEFAULT_VM_ARGUMENTS = javaOpts.toArray(new String[javaOpts.size()]);
 
         // Default JVM parameters for all modular JDKs
+        // Additions to these should include good explanations why in the relevant JIRA
+        // Keep them alphabetical to avoid the code history getting confused by reordering commits
         final ArrayList<String> modularJavaOpts = new ArrayList<>();
-        modularJavaOpts.add("--add-exports=java.base/sun.nio.ch=ALL-UNNAMED");
-        modularJavaOpts.add("--add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED");
-        modularJavaOpts.add("--add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED");
```

Comment:
@jamezp I'm not sure if I should remove these. Per any earlier comment from Richard we no longer need them, so I dropped them from the script files, JvmType etc. But perhaps I should restore these for the same reason we keep --add-modules=java.se a few lines down.

WDYT?

Reply:
From zulip:

Brian Stansberry: @James Perkins re some of our earlier discussions, should we leave old but no longer needed modular java opts in AbstractCommandBuilder?

Brian Stansberry: https://github.com/wildfly/wildfly-core/pull/4591#discussion_r638243114

James Perkins: I think we should remove them.

Brian Stansberry: cool

Brian Stansberry: so that PR is ready to go assuming CI likes it!

James Perkins: Excellent.